### ISSUE TITLE
regolith should show warning before going to sleep/suspend

### ISSUE THREAD

**Issue by tuanbass** [regolith-linux/regolith-powerd#4]
Lots of the time, I sit before my monitor without touching my keyboard, just to think about something. Or maybe I present something to my colleague.  

And it's really annoy for me, when my computer go  to sleep.

**Comment by SoumyaRanjanPatnaik** [comment]
Sorry for the late response. I suppose turning off the display a few seconds before suspending would be a better alternative, as that would be consistent with the behaviour of most other Desktop Environments, including Gnome.

**Comment by SoumyaRanjanPatnaik** [comment]
Hey. Hadn't looked at this issue in a long time. I'm guessing #14 should give the appropriate warning before by dimming, turning off display and locking the screen before suspend can occur.

**Comment by kgilmer** [comment]
I think a slow fade to black is a common way to "notify" and is suitably minimal IMO

**Comment by SoumyaRanjanPatnaik** [comment]
Agreed. COSMIC idle has this behaviour. There's no sway specific way to do this. My guess is dim to the lowest possible brightness and show an overlay to make it seem as the screen is turning off.
